python/cuml/test/conftest.py

```diff
@@ -23,6 +23,19 @@
 def pytest_configure(config):
     cp.cuda.set_allocator(None)
 
+    import rmm
```

Comment:
Will we turn this into a command line option in the future?

Reply:
Thats an interesting idea. We could make a plugin that sets the `rmm.MemoryResource` via the command line. Would be relatively easy to implement. Let me know how useful you think that would be.